workflows/templates/ZWESECUR.vtl

```diff
@@ -482,20 +491,25 @@ GROUP(&STCGRP.) +
 STCID(&AUXSTC.)
 F ACF2,REFRESH(STC)
 *
-* DEFINE ZOWE SERVER PERMISIONS ...................................
+* DEFINE XMEM SECURITY RESOURCES ..................................
 *
-* define a role holding the permissions and add &ZOWEUSER to it
+* define a role holding the permissions, add &AUXUSER and
+* &ZOWEUSER to it
 *
 SET X(ROL)
-INSERT &STCGRP. INCLUDE(&ZOWEUSER.) ROLE
+INSERT &STCGRP. INCLUDE(&AUXUSER.,&ZOWEUSER.) ROLE
```

Comment:
Here, if AUXUSER and ZOWEUSER variables are set to the same acid then the command fails with the `ACF00204 DUPLICATE DATA NOT ALLOWED IN FIELD INCLUDE` message and nothing is defined. I would suggest to split it into two commands, so the sequence would be:
```
SET X(ROL)
INSERT &STCGRP. INCLUDE(&ZOWEUSER.) ROLE
F ACF2,NEWXREF,TYPE(ROL)
CHANGE &STCGRP. INCLUDE(&AUXUSER.) ADD
```
In this case, if the variables are set to the same then only the `CHANGE` command fails with the "duplicate" message which is okay.

Reply:
@vit-tomica thank you, will do

Reply:
Please see my updates. I've also fixed some missing `&` in the comments.

Reply:
Thanks. It looks OK now.